### DISCUSSION TITLE
Use modpacks AND individual mods, possible?

### DISCUSSION THREAD

**Discussion by linthalo77** [itzg/docker-minecraft-server#3126] [Q&A]
Hey everyone,

trying to install "roguelike adventures and dungeons 2" with the CF_SLUG feature. 

The modpack installed successfully but the server can't start because it's missing a mod dependency which isn't listed in the modpack for some reason. 

I've tried using CF_SLUG for the modpack and CF_FILE for individual mods but CF_FILE gets ignored. Neither MOD_PLATFORM nor MODPACK_PLATFORM is defined, only TYPE=FORGE.

Is it possible to mix both features or do I need to install all mods listed in the modpack individually with the CF_FILE feature?

Also completely unrelated but I noticed that java8-multiarch is still getting recommended even though its deprecated and discouraged:
https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/#java-8

**Comment by itzg** [comment]
Yes you can combine both modpacks using https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/mod-platforms/auto-curseforge/ and individual mods using https://docker-minecraft-server.readthedocs.io/en/latest/mods-and-plugins/curseforge-files/
